16-slices/exercises/10-append-sort-nums/solution/main.go

```diff
@@ -24,12 +24,9 @@ func main() {
 
 	var nums []int
 	for _, s := range args {
-		n, err := strconv.Atoi(s)
-		if err != nil {
-			continue
+		if n, err := strconv.Atoi(s); err == nil {
+			nums = append(nums, n)
 		}
-
-		nums = append(nums, n)
```

Comment:
We type Go code closer to the left. This was meant to show that practice as well. Can you remove this commit?

Reply:
Can you explain what you mean by "closer to the left"?

Reply:
Done!

Reply:
I mean less scopes, and with more top to bottom reading instead of left to right reading.